applications/FluidDynamicsApplication/custom_strategies/builder_and_solvers/residualbased_block_builder_and_solver_periodic.h

```diff
@@ -134,32 +134,53 @@ class ResidualBasedBlockBuilderAndSolverPeriodic
         // Assign an Equation Id to all non-duplicate nodes
         unsigned int EqId = 0;
 
+        // Modify edge/corner periodic condtions so that one of the nodes gets EquationId
+        for (ModelPart::ConditionIterator itCond = r_model_part.ConditionsBegin(); itCond != r_model_part.ConditionsEnd(); itCond++) {
```

Comment:
```suggestion
        for (auto itCond = r_model_part.ConditionsBegin(); itCond != r_model_part.ConditionsEnd(); itCond++) {
```

And I would use it_cond to respect the style guide